web/themes/custom/votegov/templates/block/block-content--government-banner.html.twig

```diff
@@ -70,6 +73,9 @@
       </div>
     </div>
   </div>
+  {% if logged_in %}
```

Comment:
adding button here but wasnt sure where this should live... right now its just at the bottom of the page but if i need to adjust this let me know

Reply:
Move the attach_library into the conditional and move the conditional into the page.html.twig file

Reply:
There should be no changes in this file.

Reply:
moved button out of file